pkg/exchange/binance/parse.go

```diff
@@ -297,10 +298,28 @@ func ParseEvent(message string) (interface{}, error) {
 		var event MarkPriceUpdateEvent
 		err := json.Unmarshal([]byte(message), &event)
 		return &event, err
-         case "continuousKline":
-                 var event ContinuousKLineEvent
-                 err := json.Unmarshal([]byte(message), &event)
-                 return &event, err
+
+	case "continuousKline":
+		var event ContinuousKLineEvent
+		err := json.Unmarshal([]byte(message), &event)
+		return &event, err
+
+	case "ACCOUNT_UPDATE":
```

Comment:
it's all in upper case?

Reply:
Ya! It's kind of weird. I DID confirm it more than once. 

FYR: 
- [API Docs](https://binance-docs.github.io/apidocs/futures/en/#event-margin-call)

Reply:
thanks~